docs/examples/customization/external-auth-headers/echo-service.yaml

```diff
@@ -18,7 +18,7 @@ spec:
       terminationGracePeriodSeconds: 60
       containers:
       - name: echo-service
-        image: electroma/ingress-demo-echosvc-amd64:0.1
```

Comment:
Ooh did we had this?? 🤦‍♂️

Reply:
Yeah. I was surprised too. This PR will fix it.